.packit.yaml

```diff
@@ -16,11 +16,13 @@ jobs:
     - epel-7-x86_64
   trigger: pull_request
 - job: tests
-  trigger: pull_request
   metadata:
+    owner: "@oamg"
+    project: convert2rhel
```

Comment:
Is this necessary for tests? It seems like it's just for copr.

Reply:
Removed it